apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -64,8 +64,12 @@ export function setUserActiveStatus(userId: string, active: boolean, confirmReli
 			throw new Meteor.Error('user-last-owner', '', rooms);
 		}
 
-		closeOmnichannelConversations(user, livechatSubscribedRooms);
-		Promise.await(relinquishRoomOwnerships(user, chatSubscribedRooms, false));
+		Promise.await(
+			Promise.all([
```

Comment:
hopefully both actions don't have dependencies on each other 🙈

Reply:
They don't, cause they operate on different rooms each, so we should be good 👀